vtkext/private/module/vtkF3DImguiActor.cxx

```diff
@@ -176,6 +198,20 @@ struct vtkF3DImguiActor::Internals
       {
         const ImDrawCmd* cmd = &cmdList->CmdBuffer[iCmd];
 
+        // Activate texture and set uniforms per draw command:
+        vtkTextureObject* texObj = reinterpret_cast<vtkTextureObject*>(cmd->GetTexID());
+        if (!texObj)
+        {
+          texObj = this->FontTexture.Get(); // Fallback if null
```

Comment:
that cant be right, this code is generic and not font specific

Reply:
I *think* it's correct. The shader used to display draw call from imgui always use a texture. In the case of `AddLine` used to display the dashed lines in the dropzone, there is no associated texture, so I assume it's `nullptr` in this case.  
However, we need still the font atlas, so activating the font texture makes sense.